src/view/event-edit-view.js

```diff
@@ -1,123 +1,131 @@
+import dayjs from 'dayjs';
 import { createElement } from '../render.js';
+import { POINT_TYPE, CITY_NAME } from '../constans.js';
+import { OFFERS_LIST } from '../mock/offers.js';
+
+const createOfferPhotos = (photos) => photos.map(({src, description}) => (
+  `<img class="event__photo" src="${src}" alt="${description}">`)).join('');
+
+const createEventTypes = () => POINT_TYPE.map((type) => (
+  `<div class="event__type-item">
+    <input id="event-type-${type}-1" class="event__type-input  visually-hidden" type="radio" name="event-type" value="${type}">
+    <label class="event__type-label  event__type-label--${type}" for="event-type-${type}-1">${type}</label>
+  </div>`)).join('');
+
+const createEventCity = () => CITY_NAME.map((city) => (
+  `<option value="${city}"></option>`)).join('');
+
+const createOffers = (allOffersForType, currentOffersList) => allOffersForType.offers.map(({id, title, price}) => {
+  const checked = currentOffersList.includes(id) ? 'checked' : '';
+  return (
```

Comment:
```suggestion
  const checked = currentOffersList.includes(id) ? 'checked' : '';
  
  return (
```

если что-то перед ретерном есть что-то, добавляй пустую строку.